third_party/3/jwt/contrib/algorithms/__init__.pyi

```diff
@@ -0,0 +1,8 @@
+from typing import Callable, Any, Union
+from hashlib import _DataType
+
+# In reality, _HashAlg is a function of the type
+# that hashlib.sha256/384/512 are, but there doesn't
+# seem to be a consistent exportable name that we can reference
+# for that, so we'll just say it's a Callable here.
```

Comment:
Sorry, I found out I was wrong and deleted that comment, replacing it instead with "You need to merge/rebase", but I never sent that comment out. :-( So the upshot is that yes, I debugged it, and the solution is to merge from upstream.